lib/LMSManagers/LMSUserManager.php

```diff
@@ -179,6 +179,7 @@ public function getUserList($params = array())
                 accessfrom, accessto, rname, twofactorauth
             FROM vallusers
             WHERE deleted = 0'
+                . (isset($access) ? ' AND access = ' . $access : '' )
```

Comment:
Tak dla pewności - $access na zewnątrz jest walidowane?
Jeśli nie to może lepiej intval($access)?

Reply:
generalnie $access nie zmieniamy nigdzie z interfejsu ale dodałem intval() dla pewności,